src/components/modules/document/common/Upload/modules/Video/modules/Recorder/modules/VideoStream/VideoStream.view.vue

```diff
@@ -70,16 +70,46 @@ export default {
         default:
           break;
       }
+
+      this.$emit('error', null);
     },
   },
 
   methods: {
+    // eslint-disable-next-line consistent-return
```

Comment:
bad practice to disable eslint
in the end of `catch` add `return null`

Reply:
done